apis/fluentd/v1alpha1/fluentd_types.go

```diff
@@ -36,6 +36,10 @@ const (
 type FluentdSpec struct {
 	// Fluentd global inputs.
 	GlobalInputs []input.Input `json:"globalInputs,omitempty"`
+	// Select cluster filter plugins used to filter for the default cluster output
+	DefaultFilterSelector *metav1.LabelSelector `json:"defaultFilterSelector,omitempty"`
```

Comment:
What is the processing logic for this? I don't see anything in the code that handles that field.

Reply:
This is handled in the controllers/fluentdconfig_controller.go file starting from line 140.

Reply:
> This is handled in the controllers/fluentdconfig_controller.go file starting from line 140.

@Jakob3xD I cannot see how `DefaultFilterSelector`/`DefaultOutputSelector` and `default_route` is handled in controllers/fluentdconfig_controller.go